modules/vectra.py

```diff
@@ -90,7 +90,10 @@ def _generate_params(args, valid_keys, deprecated_keys):
     for k, v in args.items():
         if k in valid_keys:
             if v is not None:
-                params[k] = v
+                if k == "checkpoint":
```

Comment:
If you make this a dict, you can support future param mappings

```
   param_overrides = {"checkpoint", "from"}
   params = {}
    for k, v in args.items():
        if k in valid_keys:
            if v is not None:
                if k in param_overrides:
                    params[param_overrides[k]] = v
                else:
                    params[k] = v
```

Reply:
I have made this change. Good catch.